core/src/main/java/org/elasticsearch/index/query/EmptyQueryBuilder.java

```diff
@@ -34,10 +34,25 @@
 public final class EmptyQueryBuilder extends AbstractQueryBuilder<EmptyQueryBuilder> {
 
     public static final String NAME = "empty_query";
-
-    /** the one and only empty query builder */
     public static final EmptyQueryBuilder PROTOTYPE = new EmptyQueryBuilder();
```

Comment:
Is this still here because its not easy to get rid of or because we are planning to hopefully be able to remove the whole class soon?